pept/data/base.py

```diff
@@ -1393,6 +1408,25 @@ def __next__(self):
 
         return self._point_data[(self._index - self._sample_size):self._index]
 
+    def calculate_mean(self):
+        '''Calculate the mean standard deviation of the position over all points
+
+
+        Returns
+        --------
+        std : list of floats
+            The mean standard deviation in position in the x, y and z planes
+            Formatted as [x.mean, x.std, y.mean, y.std, z.mean, z.std]
+
+        '''
+
+        x = self.point_data[:,1] 
+        y = self.point_data[:,2]
+        z = self.point_data[:,3]
+
+        return [x.mean(), x.std(), y.mean(), y.std(), z.mean(), z.std()]
+
+
```

Comment:
Do we really need this function? Users can just use these numpy functions on their own...

Reply:
It's pretty useful to return the information quickly, especially for static data. Rather than having to type np.mean(data.point_data[0,1]) etc each time.